sdm-api-app.groovy

```diff
@@ -548,7 +550,7 @@ void deleteEventSubscription() {
 }
 
 def logToken() {
-    log.debug("Access Token: ${state.googleAccessToken}")
+    logDebug("Access Token: ${state.googleAccessToken}")
```

Comment:
intentionally left this one as `log.debug` as it is triggered only by the "Debug button", so we want to log it regardless of the toggle -- can bump it to `log.info` so it's clearer to always log?

Reply:
Understood, just thought it was typo.